### PR TITLE
Update fonts and colors to match the new design

### LINKED ISSUE #571: Optimize font loading
NOTE: This should ideally be solved as part of the upcoming redesign, to prevent duplicate work.

Currently, we preload `.ttf` font files that are not used by modern browsers (which prefer woff2 or similar):

These are preloaded from the `_document.tsx`

image

## Next steps
- See if this could be a good solution for us: https://nextjs.org/docs/app/building-your-application/optimizing/fonts

### LINKED ISSUE #590: Update fonts and colors on the website to match the new design system
This is a first step in the redesign of the website.

I will work on this since it needs close collaboration with the design team.

Part of this work will also be about cleaning up old assets, fonts and similar.

### LINKED ISSUE #610: Color blindness improvement on map
**Description**
The map currently uses a color scale with shade transitions which may be difficult to interpret for users who only see greyscale.

**Motivation**
The website should be inclusive and user-friendly for everyone, including those affected by rare forms of color blindness.

**Definition of Done**
The new color scale should correctly reflect the numeric gradient. In greyscale, the 10-15% category looks darker than the 5-10% category.
Inspiration could be taken from the [viridis color maps](https://cran.r-project.org/web/packages/viridis/vignettes/intro-to-viridis.html).

**When should it be done?**
Not urgent.

**Contact person**
@lalarus says "probably @Greenheart". 😝 

**Example**
image

### LINKED ISSUE #614: Make graph integrals transparent in new design facelift
**Description**
Make the municipality graph integrals transparent in the new design facelift (#593 ).

**Motivation**
What goals for klimatkollen would this help achieve?

**Definition of Done**
The transparency of the municipality graph integrals should be on par with their current live site appearance.

**When should it be done?**
This should be done at the same time as design facelift is released.

**Contact person**
@elvbom

### LINKED ISSUE #616: Enhance the color scheme of the municipality map in design facelift
**Description**
Color scheme for municipality map in design facelift (#593 ) should be updated.

**Motivation**
The color scheme should be more visually appealing (while still taking #610 into account).

**Definition of Done**
The color scheme should be given a thumps up by @Greenheart and @elvbom (there's no objective DoD).

**When should it be done?**
This should be done at the same time as design facelift is released.

**Contact person**
@Greenheart @elvbom

### PR DESCRIPTION
**Pull request by Greenheart**
fix #590 
fix #571 (by removing the old fonts which caused the warnings)
fix #610 (by adding using a color scale with better contrast)
fix #616 (by changing colors, and simultaneously breaking the fix for #611)
fix #614 

This is a first step before we make bigger design changes.

## Progress
- [x] Update fonts to `DM Sans Variable` 
- [x] Remove Borna
- [x] Remove roboto
- [x] Remove Source Serif
- [x] Update colors (bigger task which will require many changes)

In addition to the design updates, this PR also includes several other improvements
- Improved footer layout
- Several bug fixes
- Several UX improvements

### CODE REVIEW

File: components/RegionalView.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const InfoText = styled.div`
   position: -webkit-sticky;
   position: sticky;
   bottom: 0;
-  background: ${({ theme }) => theme.lightBlack};
+  background: ${({ theme }) => theme.newColors.black2};
   border-bottom-left-radius: 8px;
```
Comment (Greenheart) [inline components/RegionalView.tsx:30]: This needs a larger font size on tablet and up.

Also for CompanyView.

**Comment by Greenheart** [comment]
Potential bug. Investigating before we can merge.

File: public/locales/sv/startPage.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
     "scope1n2": "Egna utsläpp",
     "scope3": "I värdekedjan",
     "source": "Källa: företagens egna årsredovisningar och hållbarhetsrapporter",
-    "tableFooterInfo": "Företags utsläpp 2023 i antal ton CO₂e. [Kontakta oss](mailto:hej@klimatkollen.se) för att redigera informationen. Läs mer om [vår metod](/metodbeskrivning-foretagsdata.pdf)."
+    "tableFooterInfo": "Företags utsläpp 2023 i antal ton CO₂e. [Kontakta oss](mailto:hej@klimatkollen.se) för att redigera informationen. Läs mer om [vår metod](/metodbeskrivning-foretagsdata.pdf).",
```
Comment (elvbom) [inline public/locales/sv/startPage.json:7]: 2024!

**Review by elvbom — APPROVED** [review]
Well done!

File: components/PillSwitch.tsx
[the remark this answers is not in the document]
Comment (Greenheart) [reply components/PillSwitch.tsx:1]: We might solve this with page transitions in the future.

**Pull request merged**